src/intl/en/page-learn.json

```
@@ -1,37 +1,37 @@
+  "what-is-crypto-2": "Cryptocurrencies, such as Bitcoin, enable anyone to transfer money globally. Ethereum does too, but it can also run code that enables people to create apps and organizations. It’s both resilient and flexible: any computer program can run on Ethereum. Learn more and find out how to get started:",
   "what-is-ethereum-card-title": "What is Ethereum?",
   "what-is-ethereum-card-description": "If you are new, start here to learn why Ethereum matters.",
   "what-is-ethereum-card-image-alt": "Illustration of a person peering into a bazaar, meant to represent Ethereum.",
   "what-is-eth-card-title": "What is ETH?",
   "what-is-eth-description": "Ether (ETH) is the currency powering the Ethereum network and apps.",
-  "where-can-i-get-eth-card-title": "Where can I get ETH?",
-  "where-can-i-get-eth-card-description": "There are many ways to get ETH, depending on your location.",
+  "what-is-web3-card-title": "What is Web3?",
+  "what-is-web3-card-description": "Web3 is a model for the internet valuing ownership of your own assets and identity.",
   "additional-reading-more-on-ethereum-basics": "More on Ethereum basics",
+  "guides-hub-desc": "Guides: step-by-step instructions on how to use Ethereum",
+  "quiz-hub-desc": "Quiz hub: test your knowledge and learn more",
   "additional-reading-what-are-smart-contracts": "What are smart contracts?",
-  "additional-reading-a-developers-intro": "An introduction to Ethereum for developers",
   "additional-reading-what-is-web3": "What is web3?",
-  "additional-reading-value": "What is value?",
-  "additional-reading-decentralize-everything": "Decentralize everything",
-  "how-do-i-use-ethereum-1": "Using Ethereum is like moving to a new place. Things will seem unfamiliar at first, and everyone moves for their own reasons. Despite this, the first thing you'll need are new keys. The software that creates and holds the keys to your new home is called a \"wallet\".",
+  "additional-reading-decentralize-everything": "Video: Explanation of what Ethereum is by Vitalik Buterin",
+  "additional-reading-get-eth": "Learn how to get ETH",
+  "how-do-i-use-ethereum-1": "Using Ethereum can mean lots of things to lots of people. Maybe you want to sign in to an app, prove your online identity, participate in a decentralized community, transfer some ETH or swap some tokens. Maybe you want to build something that anyone can use. The first thing you'll need is an account. The easiest way to create and access an account is using software called a \"wallet\".",
   "what-is-a-wallet-card-title": "What is a wallet?",
-  "what-is-a-wallet-card-description": "Wallets are like keyrings: they store what you need to get access to the places you value.",
+  "what-is-a-wallet-card-description": "Digital wallets are similar to physical ones; they store what you need to prove who you are and get access to the places you value.",
```

Some brainstorming

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like physical wallets; they store what you need to prove who you are and get access to the places you value.",
```

Or maybe

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like real wallets; they store what you need to prove who you are and get access to the places you value.",
```

Second sentence

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like real wallets; they store what you need to prove your identity and get access to the places you value.",
```

Maybe too far?

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like real wallets. They keep important stuff that shows who you are and lets you use what you care about.",
```